test/eventpattern.test.ts

```diff
@@ -42,7 +42,7 @@ describe("event pattern", () => {
     test("index access", () => {
       ebEventPatternTestCase(
         reflect<EventPredicateFunction<TestEvent>>(
-          (event) => event["source"] === "lambda"
```

Comment:
Kind of defeats the purpose of the test?

Reply:
Could update to use `detail-type` (I hadn't added detail type when I created the test)

Reply:
Hmm I don't remember changing this.

Reply:
Oh, linting changed it. Didn't realize this test was on purpose using that syntax.

Reply:
Will es-ignore